org.eclipse.jdt.core/META-INF/MANIFEST.MF

```diff
@@ -6,6 +6,16 @@ Bundle-Version: 3.40.100.qualifier
 Bundle-Activator: org.eclipse.jdt.core.JavaCore
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
+Import-Package: org.eclipse.core.internal.runtime,
```

Comment:
Based on discussion at https://github.com/eclipse-equinox/equinox/discussions/697 , it looks like we can add `;bundle-symbolic-name="org.eclipse.core.runtime";bundle-version="[3.29.0, 4.0)"` to each Import-Package to keep some versioning.